ui/src/templates/constants/defaultTemplates.ts

```diff
@@ -0,0 +1,989 @@
+export const systemTemplate = () => ({
```

Comment:
will these templates eventually live on the backend as a document?

Reply:
no we are responsible for them :(